### DISCUSSION TITLE
steam: cannot run /usr/bin/bwrap

### DISCUSSION THREAD

**Discussion by variasdesign** [netblue30/firejail#6068] [Q&A] [answered]
I have the kernel parameter `kernel.unprivileged_userns_clone` set to `0`. As such, I have to use the suid version of the bubblewrap package for Steam or games won't launch. However, Firejail is not letting Steam check if `/usr/bin/bwrap` exists so it falls back to the non-suid version its runtime has. Running Steam outside Firejail works as expected. I've tried putting `noblacklist /usr/bin/bwrap` as well as ignoring [these directives](https://github.com/netblue30/firejail/discussions/5220) in my `steam.local` but it doesn't work. What am I doing wrong? Thanks for any help.

**Comment by rusty-snake** [comment, marked as answer]
Bubblewrap does not work inside firejail. See #3647.

**Reply by variasdesign** [reply]
I see, thanks for the fast response.
